scripts/reformat-all-commands.sh

```diff
@@ -23,12 +23,12 @@ CLI_TOOLS="~/go/bin/" # path to goal, algokey, etc.
 # CLI INDEXER
 ./reformat.py -doc-dir ../docs/clis/indexer/ -cmd $CLI_TOOLS/algorand-indexer
 
+# CLI CONDUIT
+./reformat.py -doc-dir ../docs/clis/conduit/ -cmd $CLI_TOOLS/conduit
```

Comment:
@Eric-Warehime this change seems to delete the /cli/conduit content but does not replace/update it when I run this script. Reverting this change allows me to build the docs as expected.

Reply:
Taking another look, it seems like this line needs to be reverted https://github.com/algorand/docs/commit/8e49cd3b1acdfe81536f7513d617626e59be4379#diff-8e1b60ec6c843b0b2386368e4a45be4c548940386a7c04928369a02b9ca53942L108

I think I had removed this to run the workflow locally, but it shouldn't have been included in the PR.

As for the `reformat.py`, I wonder which version of the indexer binaries we're grabbing in https://github.com/algorand/docs/blob/staging/.github/workflows/pr-generator.yml#L78 If it's not a version with the `conduit` command it would probably end up removing the docs.

Reply:
Let's merge it for now and will review on the other side. This script does not impact the properly generated docs in this PR.